src/shared/layout/footer/footer.tsx

```diff
@@ -7,13 +7,16 @@ export const Footer = () => {
       <div className="usa-footer__secondary-section footer">
         <div className="grid-container">
           <div className="grid-row grid-gap">
-            <div className="usa-footer__logo grid-row mobile-lg:grid-col-6 mobile-lg:grid-gap-2">
+            <div className="usa-footer__logo grid-row mobile-lg:grid-col-6 mobile-lg:grid-gap-1">
               <div className="mobile-lg:grid-col-auto">
                 <p className="usa-text-small">Powered by</p>
               </div>
               <div className="mobile-lg:grid-col-auto">
                 <img src={faasLogo} alt="footer logo" style={{ marginTop: '-20px' }} width="150" height="20" />
               </div>
+              <div className="mobile-lg:grid-col-auto">
+              <p className="usa-text-small"> | Release 1.0</p>
```

Comment:
Is there a way to read the release number from the manifest file?  Not sure if gradle works this way, but on my previous job, when we packaged a jar with maven it contained a manifest file with the version information in it - we were able to read the version from the manifest file in the code.  That way, all we had to do when upgrading a version was upgrade the version in maven, and everything else just worked.  Less to remember to update.  Not sure if we can do that here?

Reply:
Gradle and maven can not be integrated here. They are only for the backend. There is a way in react. I just hard-coded for now to test deplyment and this PR is not ready yet it is WIP.